src/Raven.Server/Documents/Sharding/ShardedDocumentDatabase.cs

```diff
@@ -226,6 +238,31 @@ public async Task DeleteBucketAsync(int bucket, long migrationIndex, string upto
         }
     }
 
+    private async Task WaitForConfirmationIndex(long confirmationIndex)
```

Comment:
ehh... method needs to end with `Async` :)

Reply:
let's add Orchestrator in the name of the method. It doesn't wait for confirmation on shards

Reply:
fixed